scipy/optimize/_differentialevolution.py

```diff
@@ -1220,7 +1220,8 @@ def solve(self):
                                   UserWarning, stacklevel=2)
             if self.disp:
                 print(f"Polishing solution with '{polish_method}'")
-            result = minimize(self.func,
+            result = minimize(lambda x:
+                                list(self._mapwrapper(self.func, np.array([x])))[0],
```

Comment:
I think I'd prefer to use `np.atleast_2d(x)` instead of `np.array([x])` because it's more explicit.